util/ai/prompt.py

```
@@ -0,0 +1,180 @@
+        ports=format_ports(image_analysis.exposed_ports),
+        env_vars=format_env_vars(image_analysis.environment_vars),
+        entrypoint=format_command_list(image_analysis.entrypoint),
+        cmd=format_command_list(image_analysis.cmd),
+        labels=format_labels(image_analysis.labels),
+        base_image=image_analysis.base_image or "Unknown",
+    )
+
+    # Ensure prompt doesn't exceed max length
+    if len(prompt) > MAX_PROMPT_LENGTH:
+        # Truncate from the layer commands section
+        overflow = len(prompt) - MAX_PROMPT_LENGTH
```

Ruff (0.14.8)

Local variable `overflow` is assigned to but never used

Remove assignment to unused variable `overflow`

(F841)